lib/commands/element.js

```diff
@@ -12,16 +12,19 @@ Object.assign(extensions, iosCommands.element);
 commands.getAttribute = async function (attribute, el) {
   el = util.unwrapElement(el);
   if (!this.isWebContext()) {
-    const value = await this.proxyCommand(`/element/${el}/attribute/${attribute}`, 'GET');
+    let value = await this.proxyCommand(`/element/${el}/attribute/${attribute}`, 'GET');
+    // Transform the result for the case when WDA returns an integer representation for a boolean value
+    if ([0, 1].indexOf(value) !== -1) {
```

Comment:
in the worst case we could also do the distinction based on the actual attribute name, but it looks odd in this particular case.